x-pack/test/apm_api_integration/tests/sourcemaps/sourcemaps.ts

```diff
@@ -4,14 +4,19 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import { unzip as unzipSync } from 'zlib';
 import pRetry from 'p-retry';
 import type { APIReturnType } from '@kbn/apm-plugin/public/services/rest/create_call_apm_api';
 import type { ApmSourceMap } from '@kbn/apm-plugin/server/routes/source_maps/create_apm_source_map_index_template';
 import type { SourceMap } from '@kbn/apm-plugin/server/routes/source_maps/route';
 import expect from '@kbn/expect';
 import { first, last, times } from 'lodash';
+import { promisify } from 'util';
+import { GetResponse } from '@elastic/elasticsearch/lib/api/types';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
 
+const unzip = promisify(unzipSync);
```

Comment:
what's the point of promisifying this?

Reply:
It's easier working with promises (async/await) than callbacks. 
Or am I missing your point?